# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/moz.build
# Commit: 33ae2aabdf22
# Full Hash: 33ae2aabdf225a72dfceb09424e4e43ae9430e69
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-06-12 16:31:09
# Regressor Bug: 1968812
# File Overlap Count: 1
# Description:
#   Revert "Bug 1968812 - Remove the CallWorker thread and use TaskQueue::Observer instead. r=bwc" for causing failures complaining about mutexLock.
#   
#   This reverts commit d53f26b530b4ad59aebe314c55a8af2c62a80fc3.
#   
#   Revert "Bug 1968812 - Copy WebrtcTaskQueueWrapper.h to WebrtcTaskQueueWrapper.cpp. r=bwc"
# ==============================================================================

diff -r ca232b0163c7 -r 33ae2aabdf22 dom/media/gtest/moz.build
--- a/dom/media/gtest/moz.build	Wed Jun 11 22:34:11 2025 +0300
+++ b/dom/media/gtest/moz.build	Wed Jun 11 23:24:24 2025 +0300
@@ -75,14 +75,9 @@
 ]
 
 if CONFIG["MOZ_WEBRTC"]:
-    LOCAL_INCLUDES += [
-        "/dom/media/webrtc/libwebrtcglue",
-    ]
-
     UNIFIED_SOURCES += [
         "TestAudioInputProcessing.cpp",
         "TestRTCStatsTimestampMaker.cpp",
-        "TestWebrtcTaskQueueWrapper.cpp",
     ]
 
 if CONFIG["OS_TARGET"] != "Android":